flashinfer/trace/__init__.py

```
@@ -17,9 +17,22 @@
+    "TraceTemplate",
+    "Var",
+    "Const",
+    "Tensor",
+    "Scalar",
+    "Solution",
+    "BuildSpec",
+    "SourceFile",
+    "SupportedLanguages",
+    "SupportedBindings",
+    "_TRACE_DUMP_DIR",
+]
```

Ruff (0.15.12)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)